.storybook/webpack.config.js

```diff
@@ -39,6 +39,10 @@ module.exports = ({ config }) => {
       {
         loader: 'ts-loader',
         options: {
+          // Temporarily ignore implicit any warnings caused by source-loader.
+          // Ignoring 7005 can be removed when the GitHub issue is resolved.
+          // See: https://github.com/storybookjs/storybook/issues/7829
+          ignoreDiagnostics: [7005],
```

Comment:
As a result of having to parse TypeScript source-loader is causing implicit any warnings on Storybook examples. This line can be removed once this [GitHub issue](https://github.com/storybookjs/storybook/issues/7829) is resolved.